lib/absinthe/phase/schema.ex

```diff
@@ -162,6 +162,23 @@ defmodule Absinthe.Phase.Schema do
     node
   end
 
+  defp set_schema_node(
+         %Blueprint.Input.Field{} = node,
+         %Blueprint.Input.Object{schema_node: %{of_type: %Absinthe.Type.Scalar{}}} = _parent,
```

Comment:
This works for my specific test but need to figure out a more generic way to handle this